next-env.d.ts note in templates

oBusk opened the pull request:
Hello! I was using `npx create-next-app --ts` to quickly bootstrap a basic next.js project with Typescript. It bothered me that I got git diff just from running `npm run build` inside the project, because the new notice in the `next-env.d.ts` file.

![image](https://user-images.githubusercontent.com/13413409/129115266-80e00bf3-78aa-40be-bd0f-a18aaa448a68.png)

So I went ahead and updated the `next-env.d.ts` file in [`packages/create-next-app/templates/typescript`](https://github.com/oBusk/next.js/blob/fb67ce9864f3561bd8883092d215960040402aa7/packages/create-next-app/templates/typescript/next-env.d.ts) to be exactly how the file looks after running once. (https://github.com/vercel/next.js/commit/7417ecc09c6b44fb4ebfdf6acd13cbcec784a44e)

Then I realized that I could probably do the same for all the `next-env.d.ts` that are spread out in the many examples, to make running those examples after cloning them feel a bit smoother. (https://github.com/vercel/next.js/commit/fb67ce9864f3561bd8883092d215960040402aa7)

> However I skipped the [`with-typescript-graphql`](https://github.com/vercel/next.js/tree/40f85f6d95b3f69421bbf13cced68754e009eed4/examples/with-typescript-graphql) example since that [`next-env.d.ts`](https://github.com/vercel/next.js/blob/40f85f6d95b3f69421bbf13cced68754e009eed4/examples/with-typescript-graphql/next-env.d.ts#L4-L9) actually has changes and I couldn't even run `yarn build` in that example, so I'm not sure what will happen with the `next-env.d.ts`. Someone who _gets_ that example will have to fix it I suppose.

ijjk commented:
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 17.6s | 17.1s | -503ms |
| buildDurationCached | 4.1s | 3.8s | -293ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.164 | 3.141 | -0.02 |
| / avg req/sec | 790.14 | 795.89 | +5.75 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 2.036 | 1.965 | -0.07 |
| /error-in-render avg req/sec | 1227.97 | 1272.02 | +44.05 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ✓ |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.13 kB | 4.13 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 491 B | 491 B | ✓ |
| Overall change | 491 B | 491 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 530 B | 530 B | ✓ |
| link.html gzip | 543 B | 543 B | ✓ |
| withRouter.html gzip | 524 B | 524 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 14.2s | 14.9s | ⚠️ +676ms |
| buildDurationCached | 5.7s | 5.5s | -123ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.108 | 3.168 | ⚠️ +0.06 |
| / avg req/sec | 804.35 | 789.07 | ⚠️ -15.28 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.978 | 2.143 | ⚠️ +0.16 |
| /error-in-render avg req/sec | 1263.86 | 1166.6 | ⚠️ -97.26 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.6 kB | 10.6 kB | ✓ |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.71 kB | 3.71 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.95 kB | 2.95 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 498 B | 498 B | ✓ |
| Overall change | 498 B | 498 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 577 B | 577 B | ✓ |
| link.html gzip | 589 B | 589 B | ✓ |
| withRouter.html gzip | 570 B | 570 B | ✓ |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

</details>
Commit: fb67ce9864f3561bd8883092d215960040402aa7

oBusk commented:
I felt like there could probably be an automated check so that templates would include the most recent version but I don't know where that would fit..

Just looking around in the repo it looks like it could fit in the [`check-examples.sh`](https://github.com/vercel/next.js/blob/40f85f6d95b3f69421bbf13cced68754e009eed4/scripts/check-examples.sh) script, but I can't see when that script is run? 

Updating the loop where it goes through every example

```bash
for folder in examples/* ; do
  cp -n packages/create-next-app/templates/default/gitignore $folder/.gitignore;
  if [ -f "$folder/package.json" ]; then
    cat $folder/package.json | jq '.license = "MIT"' | sponge $folder/package.json
  fi
  if [ -f "$folder/next-env.d.ts" ]; then
    cp packages/create-next-app/templates/typescript/next-env.d.ts $folder/next-env.d.ts
  fi
done;
```

ijjk commented:
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 14.4s | 14s | -393ms |
| buildDurationCached | 3.4s | 3.3s | -44ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.702 | 2.6 | -0.1 |
| / avg req/sec | 925.33 | 961.69 | +36.36 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.465 | 1.49 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 1705.98 | 1678.3 | ⚠️ -27.68 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ✓ |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.13 kB | 4.13 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 491 B | 491 B | ✓ |
| Overall change | 491 B | 491 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 530 B | 530 B | ✓ |
| link.html gzip | 543 B | 543 B | ✓ |
| withRouter.html gzip | 524 B | 524 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 12.5s | 12.2s | -267ms |
| buildDurationCached | 5s | 4.8s | -177ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.658 | 2.79 | ⚠️ +0.13 |
| / avg req/sec | 940.42 | 896.16 | ⚠️ -44.26 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.488 | 1.484 | 0 |
| /error-in-render avg req/sec | 1679.58 | 1684.92 | +5.34 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.6 kB | 10.6 kB | ✓ |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.71 kB | 3.71 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.95 kB | 2.95 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 498 B | 498 B | ✓ |
| Overall change | 498 B | 498 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 577 B | 577 B | ✓ |
| link.html gzip | 589 B | 589 B | ✓ |
| withRouter.html gzip | 570 B | 570 B | ✓ |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

</details>
Commit: 97f43404a7a9323c1a791e01e10fb5e8a3ae8091

ijjk commented:
## Failing test suites <!--3475261048-->
Commit: 97f43404a7a9323c1a791e01e10fb5e8a3ae8091

**test/acceptance/ReactRefreshLogBox.dev.test.js** 
-  > server-side only compilation errors

<details>
<summary>Expand output</summary>

  ● server-side only compilation errors

    ScriptTimeoutError: script timeout
      (Session info: headless chrome=92.0.4515.131)

      72 |
      73 |             // Wait for application to re-hydrate:
    > 74 |             await browser.executeAsyncScript(function () {
         |             ^
      75 |               var callback = arguments[arguments.length - 1]
      76 |               if (window.__NEXT_HYDRATED) {
      77 |                 callback()

      at Object.throwDecodedError (../node_modules/selenium-webdriver/lib/error.js:550:15)
      at parseHttpResponse (../node_modules/selenium-webdriver/lib/http.js:565:13)
      at Executor.execute (../node_modules/selenium-webdriver/lib/http.js:491:26)
          at runMicrotasks (<anonymous>)
      at Proxy.execute (../node_modules/selenium-webdriver/lib/webdriver.js:700:17)
      at Object.patch (acceptance/helpers.js:74:13)
      at Object.<anonymous> (acceptance/ReactRefreshLogBox.dev.test.js:1422:3)

</details>

ijjk commented:
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 15.6s | 15.6s | -14ms |
| buildDurationCached | 3.6s | 3.6s | -7ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.695 | 2.828 | ⚠️ +0.13 |
| / avg req/sec | 927.76 | 884.07 | ⚠️ -43.69 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.588 | 1.568 | -0.02 |
| /error-in-render avg req/sec | 1573.89 | 1594.37 | +20.48 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ✓ |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.13 kB | 4.13 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 491 B | 491 B | ✓ |
| Overall change | 491 B | 491 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 530 B | 530 B | ✓ |
| link.html gzip | 543 B | 543 B | ✓ |
| withRouter.html gzip | 524 B | 524 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 12.8s | 13s | ⚠️ +149ms |
| buildDurationCached | 5s | 5.1s | ⚠️ +9ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.796 | 2.769 | -0.03 |
| / avg req/sec | 894.1 | 902.8 | +8.7 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.576 | 1.563 | -0.01 |
| /error-in-render avg req/sec | 1585.92 | 1599.08 | +13.16 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.6 kB | 10.6 kB | ✓ |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.71 kB | 3.71 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.95 kB | 2.95 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 498 B | 498 B | ✓ |
| Overall change | 498 B | 498 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 577 B | 577 B | ✓ |
| link.html gzip | 589 B | 589 B | ✓ |
| withRouter.html gzip | 570 B | 570 B | ✓ |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

</details>
Commit: c5450e186ba430cdee6ad0acf08a855b43047c1b

oBusk commented:
I fixed (https://github.com/vercel/next.js/pull/27983/commits/c5450e186ba430cdee6ad0acf08a855b43047c1b) the `with-graphql-typescript` `.d.ts` problem.

Also, in the spirit of seatbelt _and_ airbag, I updated `create-next-app` to always copy the default `next-env.d.ts` whenever you create-next-app with an `--example` that is typescript (has `tsconfig.json`). That way, if example is added in the future that does not check in an uptodate `next-env.d.ts`, `create-next-app` will fix that for you.

ijjk commented:
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 12.5s | 12.5s | ⚠️ +37ms |
| buildDurationCached | 3s | 2.9s | -53ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | -2 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.268 | 2.264 | 0 |
| / avg req/sec | 1102.48 | 1104.43 | +1.95 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.273 | 1.283 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1964.04 | 1949.11 | ⚠️ -14.93 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ✓ |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.13 kB | 4.13 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 491 B | 491 B | ✓ |
| Overall change | 491 B | 491 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 530 B | 530 B | ✓ |
| link.html gzip | 543 B | 543 B | ✓ |
| withRouter.html gzip | 524 B | 524 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 10.5s | 10.5s | -6ms |
| buildDurationCached | 4.2s | 4.1s | -35ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | -2 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.327 | 2.319 | -0.01 |
| / avg req/sec | 1074.43 | 1077.91 | +3.48 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.288 | 1.292 | 0 |
| /error-in-render avg req/sec | 1940.65 | 1934.42 | ⚠️ -6.23 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.6 kB | 10.6 kB | ✓ |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.71 kB | 3.71 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.95 kB | 2.95 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 498 B | 498 B | ✓ |
| Overall change | 498 B | 498 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 577 B | 577 B | ✓ |
| link.html gzip | 589 B | 589 B | ✓ |
| withRouter.html gzip | 570 B | 570 B | ✓ |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

</details>
Commit: 7b2a3f3bd5347cfb14bc99b1a9f36d9f4af12448

ijjk commented:
## Failing test suites <!--3479800980-->
Commit: 7b2a3f3bd5347cfb14bc99b1a9f36d9f4af12448

**test/acceptance/ReactRefreshLogBox.dev.test.js** 
-  > server-side only compilation errors

<details>
<summary>Expand output</summary>

  ● server-side only compilation errors

    ScriptTimeoutError: script timeout
      (Session info: headless chrome=92.0.4515.131)

      72 |
      73 |             // Wait for application to re-hydrate:
    > 74 |             await browser.executeAsyncScript(function () {
         |             ^
      75 |               var callback = arguments[arguments.length - 1]
      76 |               if (window.__NEXT_HYDRATED) {
      77 |                 callback()

      at Object.throwDecodedError (../node_modules/selenium-webdriver/lib/error.js:550:15)
      at parseHttpResponse (../node_modules/selenium-webdriver/lib/http.js:565:13)
      at Executor.execute (../node_modules/selenium-webdriver/lib/http.js:491:26)
          at runMicrotasks (<anonymous>)
      at Proxy.execute (../node_modules/selenium-webdriver/lib/webdriver.js:700:17)
      at Object.patch (acceptance/helpers.js:74:13)
      at Object.<anonymous> (acceptance/ReactRefreshLogBox.dev.test.js:1422:3)

</details>

ijjk commented:
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 16.9s | 15.5s | -1.4s |
| buildDurationCached | 3.5s | 3.9s | ⚠️ +416ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.028 | 2.983 | -0.04 |
| / avg req/sec | 825.5 | 837.96 | +12.46 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.927 | 1.926 | 0 |
| /error-in-render avg req/sec | 1297.26 | 1297.72 | +0.46 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ✓ |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.13 kB | 4.13 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 491 B | 491 B | ✓ |
| Overall change | 491 B | 491 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 530 B | 530 B | ✓ |
| link.html gzip | 543 B | 543 B | ✓ |
| withRouter.html gzip | 524 B | 524 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 13.9s | 14s | ⚠️ +119ms |
| buildDurationCached | 5.1s | 5.1s | ⚠️ +49ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.219 | 3.008 | -0.21 |
| / avg req/sec | 776.76 | 831.06 | +54.3 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.97 | 1.942 | -0.03 |
| /error-in-render avg req/sec | 1269.12 | 1287.54 | +18.42 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.6 kB | 10.6 kB | ✓ |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.71 kB | 3.71 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.95 kB | 2.95 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 498 B | 498 B | ✓ |
| Overall change | 498 B | 498 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 577 B | 577 B | ✓ |
| link.html gzip | 589 B | 589 B | ✓ |
| withRouter.html gzip | 570 B | 570 B | ✓ |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

</details>
Commit: 456704294fedbddf9cbc60ecfd7abcf907a63227

styfle reviewed (approved):
Thanks!

ijjk commented:
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 14.7s | 14.5s | -246ms |
| buildDurationCached | 3.5s | 3.3s | -247ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | -2 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.904 | 2.742 | -0.16 |
| / avg req/sec | 860.79 | 911.58 | +50.79 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.756 | 1.717 | -0.04 |
| /error-in-render avg req/sec | 1423.77 | 1456.35 | +32.58 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ✓ |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.13 kB | 4.13 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 491 B | 491 B | ✓ |
| Overall change | 491 B | 491 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 532 B | 532 B | ✓ |
| link.html gzip | 544 B | 544 B | ✓ |
| withRouter.html gzip | 525 B | 525 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.3s | -399ms |
| buildDurationCached | 4.8s | 4.7s | -67ms |
| nodeModulesSize | 49.1 MB | 49.1 MB | -2 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.819 | 2.736 | -0.08 |
| / avg req/sec | 886.81 | 913.77 | +26.96 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.73 | 1.683 | -0.05 |
| /error-in-render avg req/sec | 1445 | 1485.51 | +40.51 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.6 kB | 10.6 kB | ✓ |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.71 kB | 3.71 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.95 kB | 2.95 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 498 B | 498 B | ✓ |
| Overall change | 498 B | 498 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | oBusk/next.js next-env-note-in-examples | Change |
| - | - | - | - |
| index.html gzip | 577 B | 577 B | ✓ |
| link.html gzip | 590 B | 590 B | ✓ |
| withRouter.html gzip | 570 B | 570 B | ✓ |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

</details>
Commit: 1f20f923c25b16f05c751b2519e0c4d771979635
